src/openrct2/peep/Peep.cpp

```diff
@@ -3057,55 +3330,58 @@ static void peep_update_ride_sub_state_2(rct_peep* peep){
     peep_update_ride_sub_state_2_rejoin_queue(peep, ride);
 }
 
-static void peep_update_ride_sub_state_5(rct_peep* peep){
-    Ride* ride = get_ride(peep->current_ride);
+static void peep_update_ride_sub_state_5(rct_peep * peep)
+{
+    Ride * ride = get_ride(peep->current_ride);
 
-    rct_vehicle* vehicle = GET_VEHICLE(ride->vehicles[peep->current_train]);
-    for (sint32 i = peep->current_car; i != 0; --i){
+    rct_vehicle * vehicle = GET_VEHICLE(ride->vehicles[peep->current_train]);
+    for (sint32 i = peep->current_car; i != 0; --i)
+    {
         vehicle = GET_VEHICLE(vehicle->next_vehicle_on_train);
     }
 
-    if (ride->mode != RIDE_MODE_FORWARD_ROTATION &&
-        ride->mode != RIDE_MODE_BACKWARD_ROTATION){
+    if (ride->mode != RIDE_MODE_FORWARD_ROTATION && ride->mode != RIDE_MODE_BACKWARD_ROTATION)
+    {
         if (peep->current_seat != vehicle->num_peeps)
             return;
     }
 
-    if (vehicle_is_used_in_pairs(vehicle)){
-        rct_peep* seated_peep = GET_PEEP(vehicle->peep[peep->current_seat ^ 1]);
+    if (vehicle_is_used_in_pairs(vehicle))
+    {
+        rct_peep * seated_peep = GET_PEEP(vehicle->peep[peep->current_seat ^ 1]);
         if (seated_peep->sub_state != 5)
             return;
 
         vehicle->num_peeps++;
         ride->cur_num_customers++;
 
         vehicle->friction += seated_peep->var_41;
-        invalidate_sprite_2((rct_sprite*)seated_peep);
-        sprite_move(0x8000, 0, 0, (rct_sprite*)seated_peep);
+        invalidate_sprite_2((rct_sprite *)seated_peep);
+        sprite_move(0x2000, 0, 0, (rct_sprite *)seated_peep);
 
         peep_decrement_num_riders(seated_peep);
         seated_peep->state = PEEP_STATE_ON_RIDE;
         peep_window_state_update(seated_peep);
         seated_peep->time_on_ride = 0;
-        seated_peep->sub_state = 6;
+        seated_peep->sub_state    = 6;
         peep_on_enter_or_exit_ride(seated_peep, peep->current_ride, 0);
     }
 
     vehicle->num_peeps++;
     ride->cur_num_customers++;
 
     vehicle->friction += peep->var_41;
-    invalidate_sprite_2((rct_sprite*)vehicle);
+    invalidate_sprite_2((rct_sprite *)vehicle);
 
-    invalidate_sprite_2((rct_sprite*)peep);
-    sprite_move(0x8000, 0, 0, (rct_sprite*)peep);
+    invalidate_sprite_2((rct_sprite *)peep);
+    sprite_move(0x2000, 0, 0, (rct_sprite *)peep);
```

Comment:
Erm, wait, why did you do this?!

Reply:
The function consuming this value checks for `x > 0x1fff` IIRC and `0x8000` is not a valid `sint16`

Reply:
https://github.com/OpenRCT2/OpenRCT2/blob/d7f1f13d0c6e82e0e5795942e9c80d378d975b6a/src/openrct2/world/sprite.c#L541-L545

Reply:
There is the SPRITE_LOCATION_NULL constant, which _is_ valid sint16.
I have tackled this in another PR.

Reply:
ok, thanks